### DISCUSSION TITLE
I’ve been approved but I didn't get student pack benefit

### DISCUSSION THREAD

**Discussion by yamak** [community/community#133208] [GitHub Education]
### Hi everyone,

I have been approved on 20 July but didn't get the student pack benefit yet.

**Comment by tszdabee** [comment]
@yamak It usually takes around 1-2 business days for the academic benefits of the GitHub Student Developer Pack to show up after being approved. If you still haven't received your benefits after approval within 5 days, please raise another discussion so it can be addressed.

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
